tutorial/tutorial-files/mint.shtml

```diff
@@ -0,0 +1,29 @@
+<script type="text/javascript"><![CDATA[
+class Token {
+    constructor(tokenInstance) {
+        this.props = tokenInstance
+
+    }
+
+    render() {
+        return `
+        <h2>Create new coins</h2>
+        <div class="input-group">
+            <h3>Address</h3>
+            <input type="text" id="create-address" placeholder="Enter address..." />
```

Comment:
enter address to send minted coins

Reply:
Doesn't fit my wide screen even though small font, will clarify all the same.
In the context of "mint" I presumed the learner can make that connection.